python/paddle/fluid/contrib/tests/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 file(GLOB TEST_OPS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "test_*.py")
 string(REPLACE ".py" "" TEST_OPS "${TEST_OPS}")
 
+if(APPLE OR WIN32)
+    list(REMOVE_ITEM TEST_OPS test_calibration)
+endif()
```

Comment:
remove line4-6, before line1
```
if (APPLE OR WIN32 OR NOT WITH_MKL)
   return()
endif()
```
to pass the Paddle_PrCiPython35 which runs on OPENBLAS

Reply:
oh， i saw there's another case which name is test_quantize_transpiler.py? Shall we ignore it too?
My original code just remove the test_calibration case only. Please advise.

Reply:
> remove line4-6, before line1
> 
> ```
> if (APPLE OR WIN32 OR NOT WITH_MKL)
>    return()
> endif()
> ```
> to pass the Paddle_PrCiPython35 which runs on OPENBLAS
oh， i saw there's another case which name is test_quantize_transpiler.py? Shall we ignore it too?
My original code just remove the test_calibration case only. Please advise.

Reply:
if `test_quantize_transpiler.py` could run on OPENBLAS, please remain it.

Reply:
> if `test_quantize_transpiler.py` could run on OPENBLAS, please remain it.

OK.